index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+
+  <body>
+    <header class="header">
+      <h1>Quiz App</h1>
+    </header>
+
+    <!-- QUESTION CARDS -->
+    <section class="card">
+      <div class="centered">
+        <h2 class="card__heading">What property flips the axes in flexbox?</h2>
+        <button class="card__bookmark">📕</button>
+        <button class="card__button">Show Answer</button>
```

Comment:
I really like your BEM naming (we will use it starting tomorrow!). To be more specific here, you should consider the class names`card__button-bookmark` and `card__button-answer`.